pkgs/development/libraries/belcard/default.nix

```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation rec {
   pname = "belcard";
-  version = "4.4.0";
+  version = "4.4.13";
```

Comment:
I didn't forget to update the hash, but apparently boths tags point to the same revision.